modules/cli/src/main/scala/scala/cli/commands/PackagerOptions.scala

```diff
@@ -60,7 +60,31 @@ final case class PackagerOptions(
     exitDialog: Option[String] = None,
   @Group("Windows")
   @HelpMessage("Suppress Wix ICE validation (required for users that are neither interactive, not local administrators)")
-    suppressValidation: Option[Boolean] = None
+    suppressValidation: Option[Boolean] = None,
+  @Group("Windows")
+  @HelpMessage("Path to extra WIX config content")
+  @ValueDescription("path")
+    extraConfig: List[String] = Nil,
+  @Group("Docker")
+  @HelpMessage(
+    "Building the container from base image"
+  )
+  from: Option[String] = None,
```

Comment:
Maybe prefix the docker options with `docker`, like `dockerFrom`, `dockerImageRegistry`, etc.